examples/mysql-galera/image/random.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+sed -i -e "s/^server\-id\s*\=\s.*$/server-id = ${RANDOM}/" /etc/mysql/my.cnf
```

Comment:
I need a numeric ID as opposed to string. I used a $RANDOM, which is not set unless I did it this way. I forget the details but it was one of those things I tried within entrypoint. I'll see if I can clean this up and do it from entrypoint.